src/main/java/org/broadinstitute/hellbender/utils/test/VariantContextTestUtils.java

```diff
@@ -70,10 +79,124 @@ static Object normalizeScientificNotation(final Object attribute){
         return attribute;
     }
 
+    public static VariantContext sortAlleles(final VariantContext vc, final VCFHeader header){
+        final List<Allele> originalAltAlleles = vc.getAlternateAlleles();
+        final List<Allele> sortedAltAlleles = originalAltAlleles.stream().sorted().collect(Collectors.toList());
+        final List<Allele> sortedAlleles = new ArrayList<>(vc.getNAlleles());
+
+        sortedAlleles.add(vc.getReference());
+        sortedAlleles.addAll(sortedAltAlleles);
+
+        final VariantContextBuilder result = new VariantContextBuilder(vc);
+        result.alleles(sortedAlleles);
+
+        GenotypesContext newGT = AlleleSubsettingUtils.subsetAlleles(vc.getGenotypes(),2,vc.getAlleles(),sortedAlleles,
```

Comment:
Is there a unit test for subsetAlleles that test it doing reordering?  I'm worried that someone will not realize that that's a feature it's supposed to handle.

Reply:
Reminder to self: I am going to change the contract for SubsettingUtils that asserts that behavior.